### PR TITLE
[feature] phi-3 model support

### PR DESCRIPTION
**Pull request by Pherenice1125**
Support eager attention, flash attention2 and sliding window for Phi3-mini-128k-instruct @mikecovlee

### CODE REVIEW

File: mlora/common/mix_lora.py
Comment (mikecovlee) [inline mlora/common/mix_lora.py:1]: plz remove the comments. this file should not be modified.

File: model_archi.txt
Comment (mikecovlee) [inline model_archi.txt:1]: this file should not included in pr. plz remove it.

File: 1.json
Comment (mikecovlee) [inline 1.json:1]: remove this test file.

File: 2.json
Comment (mikecovlee) [inline 2.json:1]: remove this test file.

**Review by mikecovlee — CHANGES_REQUESTED** [review]

File: mlora/common/mix_lora.py
Comment (Pherenice1125) [reply mlora/common/mix_lora.py:1]: Already removed.

File: model_archi.txt
Comment (Pherenice1125) [reply model_archi.txt:1]: Already removed.

File: 1.json
Comment (Pherenice1125) [reply 1.json:1]: Already removed.

File: 2.json
Comment (Pherenice1125) [reply 2.json:1]: Already removed.

**Comment by mikecovlee** [comment]
ARC-C + LoRA = 84.2%

**Review by mikecovlee — APPROVED** [review]

**Pull request merged**